pytest_fixtures/sys_fixtures.py

```diff
@@ -21,3 +23,12 @@ def relax_bfa(default_sat):
     """Relax BFA protection against failed login attempts"""
     if default_sat:
         default_sat.cli.Settings.set({'name': 'failed_login_attempts_limit', 'value': '0'})
+
+
+@pytest.fixture(autouse=True, scope='session')
+def proxy_port_range(default_sat):
+    """Assigns port range for fake_capsules"""
+    port_pool_range = settings.fake_capsules.port_range
+    port_range = port_pool_range.replace(", ", "-")
+    if default_sat.execute(f'semanage port -l | grep {str(port_range)}').status != 0:
```

Comment:
I think no need to convert to `str` here ?

Reply:
yes, no need for str here. It was left by mistake. I will update it